qa/tasks/ceph_manager.py

```diff
@@ -2716,6 +2729,12 @@ def dump_pgs_not_active(self):
              self.log('PG %s is not active' % pg['pgid'])
              self.log(pg)
 
+    def dump_pgs_not_active_peered(self, pgs):
+        for pg in pgs:
+            if (not pg['state'].count('active')) and (not pg['state'].count('peered')):
+                self.log('PG %s is not active or peered' % pg['pgid'])
+                self.log(pg)
+
```

Comment:
@ljflores 
For me, it is easier to look at each pg stats individually rather than in a big list. Also, I was following the convention that has been implemented before me: https://github.com/ceph/ceph/blob/main/qa/tasks/ceph_manager.py#L2689-L2717

Let me know what you think!